config/locales/es.yml

```diff
@@ -2173,7 +2173,7 @@ es:
           title: Declara tus impuestos estatales de Maryland sin costo
         nc:
           closed_html: |
-            <strong>FileYourStateTaxes</strong> se creó en asociación con el estado de North Carolina para integrarse con la herramienta Direct File del IRS.<br /><br />
+            <strong>FileYourStateTaxes</strong> ha sido creado por Code for America en colaboración con el estado de [state name] para integrarse con  el servicio Direct File del IRS.<br /><br />
```

Comment:
this seems wrong -- we need to use variable here, unless this is a correct way to include a variable into a html string? I thought it should be `%{state_name}`

Reply:
extra space after con

Reply:
changing it to "Carolina del Norte" since %{state_name} isn't in the english version